waiter/config-full.edn

```diff
@@ -107,10 +107,16 @@
 
  ; ---------- Node ----------
 
+ ; The fully qualified domain name for this router
+ :router-fqdn "route1.waiter.example.com"
+
  ; Use a descriptive name for this router; this is useful in multi-router
  ; scenarios for identifying which router handled a request:
  :router-id-prefix "router-1"
 
+ ; The ssl port configured with certs for the name given by :router-fqdn
+ :router-ssl-port 12323
```

Comment:
Let's move this to `server-options` or `zookeeper`?

Reply:
Will move inside `:router-config`